NitroxLauncher/App.xaml.cs

```diff
@@ -30,6 +30,31 @@ protected override void OnStartup(StartupEventArgs e)
             {
                 MessageBox.Show("Nitrox launcher should not be executed from a temporary directory. Install Nitrox launcher properly by extracting ALL files and moving these to a dedicated location on your PC.", "Invalid working directory", MessageBoxButton.OK, MessageBoxImage.Error);
                 Environment.Exit(1);
+                Shutdown();
+            }
+
+            if (!RoleDetection.isAppRunningInAdmin()) {
+
+                if (Directory.GetCurrentDirectory().StartsWith(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), StringComparison.OrdinalIgnoreCase))
```

Comment:
This should check for subnautica install path. Not the launcher working directory. If Subnautica is installed in Program Files then the launcher likely needs admin.